src/web3_utils.py

```diff
@@ -0,0 +1,178 @@
+import logging
+
+import requests
+from hexbytes import HexBytes
+from web3 import Web3
+from web3 import contract
+from web3 import exceptions
+
+from config.constants import MULTICHAIN_CONFIG
+from src.utils import get_abi
```

Comment:
should we move get_abi and get_explorer into web3_utils? interacting with web3 objects

Reply:
I was thinking that web3 implies usage of `.web3` node, not just everything related to web3